django/contrib/admin/static/admin/css/base.css

```diff
@@ -887,6 +891,10 @@ a.deletelink:focus, a.deletelink:hover {
     width: 100%;
 }
 
+#content-main:not(.app-list) a:not([role="button"]) {
```

Comment:
```suggestion
a:not(#nav-sidebar a):not(#content-main.app-list a):not([role="button"]) {
```
I think maybe we can do this and then remove the other css rule and the `<div id="content-main">` additions.
Part of the reason is I feel like the breadcrumbs and the recent actions section should also have underlines :thinking: 

After that, I think we're just missing a review from the accessibility team

Reply:
Thank you for the great suggestion, @sarahboyce 😃

Hmm... I'm not sure if breadcrumbs really need underlines. Since breadcrumbs are already designed in a very intuitive way, I believe users can easily recognize them as links without the need for underlines.
(Considering that the target users are developers with some knowledge of the web...)
Or maybe it would be good to add a --header-link-color underline to the breadcrumbs, similar to the one applied to user-tools.
<img width="236" alt="Screenshot 2025-01-31 at 10 06 03 AM" src="https://github.com/user-attachments/assets/a7e126de-14e1-4f33-a278-18f76da78a4b" />

For now, I'll follow your suggestion and add underlines to both the breadcrumbs and recent actions.

Additionally, links are also being added to the header, which I assume was unexpected.
<img width="276" alt="Screenshot 2025-01-31 at 10 03 20 AM" src="https://github.com/user-attachments/assets/536f7292-a417-46c6-8718-343529f2ef4e" />
So, I'll exclude the links in the header and apply the following selectors instead.
```
a:not(#header a):not(#nav-sidebar a):not(#content-main.app-list a):not([role="button"]) {
    text-decoration: underline;
}
```

Reply:
I don't have a strong opinion on the breadcumb underlines - maybe I'd weakly prefer it.

But one thing to keep in mind is that the admin isn't only used by developers, it's also used a lot by customer support, operations and all sorts of people in early stage (and later) companies that haven't built their own admin yet (or never will), and I've also seen this with agencies giving the site owner who knows more or less nothing access and a bit of training to make changes to the site.